### PR TITLE
Add support for sauce connect as alternative to local selenium server

### PR DESCRIPTION
**Pull request by jefftougas**
Let me know your thoughts on this - I wanted to have the ability to switch back and forth between running WD/mocha tests locally and in sauce labs (e.g. for development vs. continuous integration), so I modified the mocha-selenium task to support either or.

I've held off on making the relevant changes to the README file or writing any tests because I wanted to get your thoughts on this approach first - does this seem reasonable to you? I considered putting this functionality in selenium-launcher itself, but this made more sense to me.

### CODE REVIEW

File: tasks/mocha-selenium.js [hunk 1]
```diff
@@ -77,12 +118,14 @@ module.exports = function(grunt) {
       var remote = options.usePromises ? 'promiseRemote' : 'remote';
       remote = options.useChaining ? 'promiseChainRemote' : remote;
 
-      var browser = wd[remote](selenium.host, selenium.port);
+      var browser = wd[remote](selenium.host, selenium.port, selenium.username, selenium.password);
```
Comment (wookiehangover) [inline tasks/mocha-selenium.js:121]: Will `wd` ignore username and password if they're undefined?

**Comment by wookiehangover** [comment]
Thanks for the pull request @jefftougas!

I'm happy with this approach--please add tests and feel free to make updates to the readme and I'll get this merged and released.

**Comment by wookiehangover** [comment]
bump @jefftougas

**Comment by jefftougas** [comment]
Sorry for the delay @wookiehangover - hoping to get the tests and readme updates over to you by this weekend.

**Pull request closed without merging**